### DISCUSSION TITLE
Cannot open repository page when the repository name is "xxx.github.io"

### DISCUSSION THREAD

**Discussion by xitieshiz2** [community/community#22504] [Other Feature Feedback, Questions, & Ideas]
<p>If I try to view any repository page when the repository name is “<a href="http://xxx.github.io" rel="noopener nofollow ugc">xxx.github.io</a>” , there’s only github pages 404 page .<br>
Such as my github pages repository <a href="https://github.com/xitieshiz/xitieshiz.github.io/" class="inline-onebox" rel="noopener nofollow ugc">GitHub - xitieshiz/xitieshiz.github.io</a><br>
And when I try to push , it simply says “The repository does not seem to exist anymore. You may not have access, or it may have been deleted or renamed.”<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/3/e3894fa6b1ce5a34250bb2614da74210728782e6.png" data-download-href="/uploads/short-url/wsSzyIgL8PomvmqU51nyp1h8O4S.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75575551/181071532-e30eed6c-3186-4b86-b7d2-f307bfdd8f29.png" alt="image" data-base62-sha1="wsSzyIgL8PomvmqU51nyp1h8O4S" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3894fa6b1ce5a34250bb2614da74210728782e6_2_690x361.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3894fa6b1ce5a34250bb2614da74210728782e6_2_1035x541.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/3/e3894fa6b1ce5a34250bb2614da74210728782e6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3894fa6b1ce5a34250bb2614da74210728782e6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1191×624 23.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Gisgar3** [comment]
<p>Hey there, welcome to the <strong>GitHub Support Community</strong>! I tried viewing it on my end and the repository seems to display fine. Are you still having issues?</p>

**Comment by xitieshiz2** [comment, marked as answer]
<p>I’m sorry , it’s solved .<br>
It’s a bug of a network software</p>

**Comment by Gisgar3** [comment]
<p>All good! Just make sure this post is marked as solved :slight_smile:</p>
